releasenotes/notes/remove-typing-marker-2b1684ceb542b51a.yaml

```
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Even though ``qiskit-algorithms`` included a ``py.typed`` marker, reporting
+    itself as being typed, the types were not checked and contained many errors.
+    This will be fixed in the 0.3 release but we are releasing a bugfix release
+    in an untyped form to allow dependent packages that are themselves typed to
+    continue type checking normally.
```

```suggestion
    Even though ``qiskit-algorithms`` included a ``py.typed`` marker, reporting
    itself as being typed, the types were not checked and contained errors.
    We are releasing a bugfix release
    in an untyped form to allow dependent packages, that are themselves typed, to
    continue type checking normally.
```
I think I would leave it more open rather than promise when this might be reverted back at this point